src/runtime/audience-action-local-flow.ts

```diff
@@ -442,22 +484,31 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
       .replace('$REWARDED_AD_CLOSE_BUTTON_HTML$', closeHtml)
       .replace('$ICON$', icon)
       .replace('$MESSAGE$', message)
-      .replace('$VIEW_AN_AD$', viewad)
+      .replace('$VIEW_AN_AD$', prioritySwaped ? support : viewad)
```

Comment:
It looks like it's basically swapping the two button elements. Would it be possible to make 2 placeholders for where the buttons are displayed and dynamically placing the entire button elements w/o having to cross-bind callback to each other?
If it requires too much work for refactoring, leaving a TODO can be okay for a temporary experiment.

Reply:
Added a todo. But because this is a temporary experiment, I think it's more likely we end up removing it before refactoring.